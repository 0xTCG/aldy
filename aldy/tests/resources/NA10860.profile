CFTR:
  down: [0]
  e1: [0]
  e10: [0]
  e11: [0]
  e12: [0]
  e13: [0]
  e14: [0]
  e15: [0]
  e16: [0]
  e17: [0]
  e18: [0]
  e19: [0]
  e2: [0]
  e20: [0]
  e21: [0]
  e22: [0]
  e23: [0]
  e24: [0]
  e25: [0]
  e26: [0]
  e27: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i10: [0]
  i11: [0]
  i12: [0]
  i13: [0]
  i14: [0]
  i15: [0]
  i16: [0]
  i17: [0]
  i18: [0]
  i19: [0]
  i2: [0]
  i20: [0]
  i21: [0]
  i22: [0]
  i23: [0]
  i24: [0]
  i25: [0]
  i26: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  i9: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
COMT:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP1A1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP1A2:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP2A13:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  up: [0]
  utr3: [0]
CYP2A6:
  down: [0, 0]
  e1: [0, 0]
  e2: [0, 0]
  e3: [0, 0]
  e4: [0, 0]
  e5: [0, 0]
  e6: [0, 0]
  e7: [0, 0]
  e8: [0, 0]
  e9: [0, 0]
  i1: [0, 0]
  i2: [0, 0]
  i3: [0, 0]
  i4: [0, 0]
  i5: [0, 0]
  i6: [0, 0]
  i7: [0, 0]
  i8: [0, 0]
  up: [0, 0]
  utr3: [0, 0]
  utr5: [0, 0]
CYP2B6:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP2C19:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP2C8:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP2C9:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP2D6:
  e1: [5773, 4876]
  e2: [7614, 4775]
  e3: [5943, 4655]
  e4: [8663, 3342]
  e5: [8894, 5152]
  e6: [6165, 4395]
  e7: [11864, 2538]
  e8: [4121, 4153]
  e9: [5298, 6829]
  i1: [27006, 21449]
  i2: [23855, 13477]
  i3: [4053, 1914]
  i4: [24528, 9574]
  i5: [7952, 6407]
  i6: [9447, 5216]
  i7: [21544, 8546]
  i8: [2942, 3288]
  ins: [11289, 22153]
  pce: [0, 73016]
  rep: [82361, 101987]
  up: [231916, 142157]
  utr3: [1630, 2608]
  utr5: [4300, 2899]
CYP2E1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  utr3: [0]
  utr5: [0]
CYP2F1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP2J2:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP2R1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP2S1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP2W1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP3A4:
  down: [0]
  e1: [0]
  e10: [0]
  e11: [0]
  e12: [0]
  e13: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i10: [0]
  i11: [0]
  i12: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  i9: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP3A43:
  down: [0]
  e1: [0]
  e10: [0]
  e11: [0]
  e12: [0]
  e13: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i10: [0]
  i11: [0]
  i12: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  i9: [0]
  up: [0]
CYP3A5:
  down: [0]
  e1: [0]
  e10: [0]
  e11: [0]
  e12: [0]
  e13: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i10: [0]
  i11: [0]
  i12: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  i9: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP3A7:
  down: [0]
  e1: [0]
  e10: [0]
  e11: [0]
  e12: [0]
  e13: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i10: [0]
  i11: [0]
  i12: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  i9: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP4F2:
  down: [0]
  e1: [0]
  e10: [0]
  e11: [0]
  e12: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i10: [0]
  i11: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  i9: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
DPYD:
  down: [0]
  e1: [0]
  e10: [0]
  e11: [0]
  e12: [0]
  e13: [0]
  e14: [0]
  e15: [0]
  e16: [0]
  e17: [0]
  e18: [0]
  e19: [0]
  e2: [0]
  e20: [0]
  e21: [0]
  e22: [0]
  e23: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i10: [0]
  i11: [0]
  i12: [0]
  i13: [0]
  i14: [0]
  i15: [0]
  i16: [0]
  i17: [0]
  i18: [0]
  i19: [0]
  i2: [0]
  i20: [0]
  i21: [0]
  i22: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  i9: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
G6PD:
  down: [0]
  e1: [0]
  e10: [0]
  e11: [0]
  e12: [0]
  e13: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i10: [0]
  i11: [0]
  i12: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  i9: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
GSTM1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
GSTP1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
IFNL3:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
NAT1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  i1: [0]
  i2: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
NAT2:
  down: [0]
  e1: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
NUDT15:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  i1: [0]
  i2: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
SLCO1B1:
  down: [0]
  e1: [0]
  e10: [0]
  e11: [0]
  e12: [0]
  e13: [0]
  e14: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i10: [0]
  i11: [0]
  i12: [0]
  i13: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  i9: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
TPMT:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
UGT1A1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
UGT2B7:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  up: [0]
VKORC1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  i1: [0]
  i2: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
neutral:
  hg19: ['22', 42547463, 42548249]
  value: 19257

