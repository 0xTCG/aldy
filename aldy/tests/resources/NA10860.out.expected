#Sample	Gene	SolutionID	Major	Minor	Copy	Allele	Location	Type	Coverage	Effect	dbSNP	Code	Status
#Solution 1: *(1.018 +rs111564371 +rs112568578 +rs113889384 +rs28371713 +rs28371701), *4.001, *(4.1021 -rs28371738)
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	0	1.018	42524707	T>C	11	none	rs111564371	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	0	1.018	42524712	C>G	12	none	rs112568578	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	0	1.018	42524742	G>A	14	none	rs113889384	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	0	1.018	42524794	A>G	14	none	rs28371713	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	0	1.018	42526048	G>C	23	none	rs28371701	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	0	1.018	42526523	G>A	11	none	rs29001678	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	1	4.001	42522391	G>A	14	none	rs28371738	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	1	4.001	42522612	C>G	15	S486T	rs1135840	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	1	4.001	42523210	T>C	34	none	rs2004511	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	1	4.001	42523408	T>G	59	none	rs1985842	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	1	4.001	42524695	T>C	34	none	rs2267447	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	1	4.001	42524946	C>T	32	splice defect	rs3892097	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	1	4.001	42525131	C>G	24	none	rs1058164	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	1	4.001	42525797	G>C	35	none	rs28371705	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	1	4.001	42525810	T>C	34	H94R	rs28371704	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	1	4.001	42525820	G>T	33	L91M	rs28371703	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	1	4.001	42525951	A>C	23	none	rs28371702	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	1	4.001	42526048	G>C	23	none	rs28371701	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	1	4.001	42526483	C>A	31	none	rs28371699	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	1	4.001	42526693	G>A	18	P34S	rs1065852	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	1	4.001	42527792	C>T	30	none	rs1080989	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	1	4.001	42528027	T>C	16	none	rs28735595	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	1	4.001	42528223	G>A	30	none	rs28588594	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	2	4.1021	42523210	T>C	34	none	rs2004511	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	2	4.1021	42523408	T>G	59	none	rs1985842	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	2	4.1021	42523527	C>T	21	R365H	rs1058172	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	2	4.1021	42524695	T>C	34	none	rs2267447	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	2	4.1021	42524946	C>T	32	splice defect	rs3892097	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	2	4.1021	42525131	C>G	24	none	rs1058164	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	2	4.1021	42525797	G>C	35	none	rs28371705	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	2	4.1021	42525810	T>C	34	H94R	rs28371704	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	2	4.1021	42525820	G>T	33	L91M	rs28371703	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	2	4.1021	42525951	A>C	23	none	rs28371702	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	2	4.1021	42526048	G>C	23	none	rs28371701	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	2	4.1021	42526483	C>A	31	none	rs28371699	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	2	4.1021	42526693	G>A	18	P34S	rs1065852	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	2	4.1021	42527792	C>T	30	none	rs1080989	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	2	4.1021	42528027	T>C	16	none	rs28735595	
NA10860	CYP2D6	1	*1/*4+*4.021.ALDY	1.018;4.001;4.1021	2	4.1021	42528223	G>A	30	none	rs28588594	
#Solution 2: *(1.018 +rs111564371 +rs112568578 +rs113889384 +rs28371713 +rs28371701), *4.021, *(4.1013 -rs28371738)
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	0	1.018	42524707	T>C	11	none	rs111564371	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	0	1.018	42524712	C>G	12	none	rs112568578	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	0	1.018	42524742	G>A	14	none	rs113889384	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	0	1.018	42524794	A>G	14	none	rs28371713	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	0	1.018	42526048	G>C	23	none	rs28371701	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	0	1.018	42526523	G>A	11	none	rs29001678	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	1	4.021	42522391	G>A	14	none	rs28371738	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	1	4.021	42522612	C>G	15	S486T	rs1135840	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	1	4.021	42523210	T>C	34	none	rs2004511	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	1	4.021	42523408	T>G	59	none	rs1985842	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	1	4.021	42523527	C>T	21	R365H	rs1058172	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	1	4.021	42524695	T>C	34	none	rs2267447	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	1	4.021	42524946	C>T	32	splice defect	rs3892097	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	1	4.021	42525131	C>G	24	none	rs1058164	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	1	4.021	42525797	G>C	35	none	rs28371705	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	1	4.021	42525810	T>C	34	H94R	rs28371704	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	1	4.021	42525820	G>T	33	L91M	rs28371703	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	1	4.021	42525951	A>C	23	none	rs28371702	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	1	4.021	42526048	G>C	23	none	rs28371701	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	1	4.021	42526483	C>A	31	none	rs28371699	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	1	4.021	42526693	G>A	18	P34S	rs1065852	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	1	4.021	42527792	C>T	30	none	rs1080989	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	1	4.021	42528027	T>C	16	none	rs28735595	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	1	4.021	42528223	G>A	30	none	rs28588594	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	2	4.1013	42523210	T>C	34	none	rs2004511	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	2	4.1013	42523408	T>G	59	none	rs1985842	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	2	4.1013	42524695	T>C	34	none	rs2267447	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	2	4.1013	42524946	C>T	32	splice defect	rs3892097	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	2	4.1013	42525131	C>G	24	none	rs1058164	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	2	4.1013	42525797	G>C	35	none	rs28371705	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	2	4.1013	42525810	T>C	34	H94R	rs28371704	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	2	4.1013	42525820	G>T	33	L91M	rs28371703	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	2	4.1013	42525951	A>C	23	none	rs28371702	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	2	4.1013	42526048	G>C	23	none	rs28371701	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	2	4.1013	42526483	C>A	31	none	rs28371699	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	2	4.1013	42526693	G>A	18	P34S	rs1065852	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	2	4.1013	42527792	C>T	30	none	rs1080989	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	2	4.1013	42528027	T>C	16	none	rs28735595	
NA10860	CYP2D6	2	*1/*4.021+*4N.ALDY	1.018;4.021;4.1013	2	4.1013	42528223	G>A	30	none	rs28588594	
